tests/phpunit/tests/comment/metaCache.php

```diff
@@ -56,19 +55,59 @@ public function test_update_comment_meta_cache_true() {
 		// Clear comment cache, just in case.
 		clean_comment_cache( $comment_ids );
 
-		$q = new WP_Comment_Query(
+		$num_queries = get_num_queries();
+		$q           = new WP_Comment_Query(
 			array(
 				'post_ID'                   => $p,
 				'update_comment_meta_cache' => true,
 			)
 		);
+		$this->assertSame( $num_queries + 2, get_num_queries() );
 
-		$num_queries = $wpdb->num_queries;
+		$num_queries = get_num_queries();
 		foreach ( $comment_ids as $cid ) {
 			get_comment_meta( $cid, 'foo', 'bar' );
 		}
 
-		$this->assertSame( $num_queries, $wpdb->num_queries );
+		$this->assertSame( $num_queries + 1, get_num_queries() );
```

Comment:
As above, this test change will need:

* messages for the multiple assertions
* assert the difference in comment count (ie 2 and 1 respectively) as it make the test outcome clearer.